Dockerfile.centos.patch

```diff
@@ -0,0 +1,31 @@
+diff --git a/Dockerfile b/Dockerfile
```

Comment:
Why patch the dockerfile and not simply provide a `Dockerfile.local` or the like since `Dockerfile` is the source of truth for the downstream for the time being

Reply:
That's exactly why. Dockerfile is the source of truth, we create Dockerfile.local using a patch. The default for local builds is Dockerfile.patch as before, this change allows you to select an alternate patch for a centos base with `PATCH=Dockerfile.centos.patch make ...`